### PR TITLE
fix: onboarding text layout

### LINKED ISSUE #1840: [Bug]: "Send bitcoin to your.." text can be wrapped undesirably on some device displays
### Describe the bug

With an empty balance, viewing the savings or spending assets page displays either "SEND BITCOIN TO YOUR SPENDING BALANCE." or "SEND BITCOIN TO YOUR SAVINGS BALANCE.". On come devices such as Galaxy 24 Ultra, Galaxy Node 10 and One Plus 9 the text wraps in a way that doesn't match the design spec.

### Reproduce

1. Using an affected device such as Samsung Galaxy 24 Ultra (can use lambdatest)
1. Without any savings or spending balance
2. On the main screen, tap on 'Savings' or 'Spending' asset
3. Observe text is wrapped in ways that can break up words or at least move the full stop to a new line(see screenshots for examples)

### Screenshots / Recording

**Galaxy Note 10**

-----

**Galaxy S24 Ultra & OnePlus 9**

### Operating system

Android 10 & 11 & 14

### Bitkit version

4ee19c9ad0693eebef7d21f1ee470a1c29f29006

### Log output

_No response_

### LINKED ISSUE #1859: [Bug]: 'transfer_successful' displayed as header
### Describe the bug

When successfully establishing a LN connection, the header of the page reads 'transfer_successful'. Looks like matching localisation text is missing; surprised there is nothing to catch this.

### Reproduce

1. Go to create a new LN channel
2. Wait a block for it to complete
3. Observe header of the success page

### Screenshots / Recording

Screenshot 2024-05-22 at 13 35 40

### Operating system

Android 13 TKQ1.221114.001

### Bitkit version

658b9f2bd22d2ca63893a89dc56f47a054546d79

### Log output

_No response_

### PR DESCRIPTION
**Pull request by limpbrains**
### Description

- Do not limit text width
- if Arrow can't fit, move it below the text
- move line breaking logic to language file
- also fixes missing nav title

### Linked Issues/Tasks

closes #1840
closes #1859

### Type of change

Bug fix

### Tests

No test

### CODE REVIEW

File: src/utils/i18n/locales/en/onboarding.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
 	"restore_failed_text": "Bitkit restored your savings, but failed to restore your spending balance and data.",
 	"restore_no_backup_button": "Proceed Without Backup",
 	"restore_no_backup_warn": "If you previously had a lightning backup it will be overwritten and lost. This could result in a loss of funds.",
-	"empty_wallet": "To get started <accent>send Bitcoin</accent> to your wallet.",
+	"empty_wallet": "To get\nstarted\n<accent>send\nBitcoinnBitcoin</accent>\nto your\nwallet.",
```
Comment (pwltr) [inline src/utils/i18n/locales/en/onboarding.json]:
```suggestion
	"empty_wallet": "To get\nstarted\n<accent>send\nBitcoin</accent>\nto your\nwallet.",
```

File: src/screens/Lightning/Success.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ const Success = ({
 
 	return (
 		<InfoScreen
-			navTitle={t('transfer_successful')}
+			navTitle={t('transfer.nav_title')}
```
Comment (pwltr) [inline src/screens/Lightning/Success.tsx]: Title should be "Transfer Successful" (typo in Figma):
image

File: src/components/WalletOnboarding.tsx [hunk 3]
```diff
@@ -36,8 +38,26 @@ const WalletOnboarding = ({
 		return [styles.root, { marginBottom: 105 + insets.bottom }];
 	}, [insets.bottom]);
 
+	const handleTextLayout = (e: any): void => {
```
Comment (pwltr) [inline src/components/WalletOnboarding.tsx]: Please avoid `any` if possible: `(e: LayoutChangeEvent)`

**Review by pwltr — CHANGES_REQUESTED** [review]

**Review by pwltr — APPROVED** [review]

**Comment by catch-21** [comment]
Tested on v118
Mostly LGTM, although I did see the arrow behind text for Russian so we probably should hide to when text is wider: https://github.com/synonymdev/bitkit/issues/1895

**Pull request merged**